chains/ethereum/connection_test.go

```diff
@@ -22,17 +22,17 @@ var Alice = keystore.TestKeyRing.EthereumKeys[keystore.AliceKey]
 
 var TestAddress = ethcmn.HexToAddress("34c59fBf82C9e31BA9CBB5faF4fe6df05de18Ad4")
 var TestAddress2 = ethcmn.HexToAddress("0a4c3620AF8f3F182e203609f90f7133e018Bf5D")
-var TestCentrifugeContractAddress = ethcmn.HexToAddress("70486404e42d17298c57b046Aa162Dc3aCc075f0")
-var TestReceiverContractAddress = ethcmn.HexToAddress("705D4Fa884AF2Ae59C7780A0f201109947E2Bf6D")
-var TestEmitterContractAddress = ethcmn.HexToAddress("60F9363AaF4993ABA818D5438db5E64bCe6E612b")
+var TestBridgeContractAddress = ethcmn.HexToAddress("0x3167776db165D8eA0f51790CA2bbf44Db5105ADF")
+var TestReceiverContractAddress = ethcmn.HexToAddress("0x5842B333910Fe0BfA05F5Ea9F1602a40d1AF3584")
+var TestEmitterContractAddress = ethcmn.HexToAddress("0x3c747684333605408F9A4907DA043ee4c1A72D9c")
```

Comment:
can we remove the contracts not being used, try and remove confusion for the future. 

Emitter and Receiver should not be used, also i'm not sure what `TestAddress` or `TestAddresss2` are